programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,5 @@
+function solution(num) {
+    return num  % 2 === 0 ? "Even" : "Odd";
```

Comment:
```suggestion
    return num  % 2 ? "Odd" : "Even";
```
0을 false로 인식하기 때문에 === 0 없이 순서를 바꾸는 것으로 조금 더 간결하게 짜실 수 있습니다.